flowzone.yml

```diff
@@ -1027,6 +1035,23 @@ jobs:
         with:
           node-version: 18.x
 
+      # https://github.com/rhysd/actionlint/blob/main/docs/usage.md#problem-matchers
+      - name: Check workflow files
+        run: |
+          set -xe
+
+          echo "::add-matcher::.github/actionlint-matcher.json"
```

Comment:
This file only exists in the Flowzone repo, so this step will always fail on all other repos.

I'll move this check to the Flowzone `tests.yml` workflow for now so we don't break all other repos.